<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainDialog</class>
 <widget class="QDialog" name="mainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainDialog{	border-image: url(C:/Users/lenovo/Desktop/mainbackground.png);}
#mainDialog *{border-image:url();}</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>256</width>
     <height>256</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url()</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="now_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>sda </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="now_signature">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>299</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url();
font: 75 12pt &quot;Adobe 宋体 Std L&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="now_rank">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Edit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>170</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PvP">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>470</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
color: rgb(255, 255, 255);
font: 75 italic 20pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>PvP</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="PvE">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>PvE</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="RankList">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Rank List</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(C:/Users/lenovo/Desktop/stop.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playing">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(C:/Users/lenovo/Desktop/do.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(C:/Users/lenovo/Desktop/pre.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(C:/Users/lenovo/Desktop/next.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="music_name">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>350</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url();
font: italic 16pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(C:/Users/lenovo/Desktop/loading.jpg)
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>340</y>
     <width>161</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="itemloop">
      <property name="styleSheet">
       <string notr="true">border-image:url();
font: 75 14pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>单曲循环</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="itemonce">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>14</pointsize>
        <weight>9</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url();
font: 75 14pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>当前一次</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="listloop">
      <property name="styleSheet">
       <string notr="true">border-image:url();
font: 75 14pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>列表循环</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="sequential">
      <property name="styleSheet">
       <string notr="true">border-image:url();
font: 75 14pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>顺序播放</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="random">
      <property name="styleSheet">
       <string notr="true">border-image:url();
font: 75 14pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>随机播放</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="RankList_2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>520</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	border-image: url();
font: 75 italic 20pt &quot;Adobe Arabic&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="music_author">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url();
font: italic 16pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RankList_2</sender>
   <signal>clicked()</signal>
   <receiver>mainDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>623</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
